Room integrations and other customizations

davideciarmiello opened the pull request:
Hi friend. On my branch i have added this features:

Show entities attributes in home assistant (used for filters in customizations )
room name retrieval for entities, used for customization friendly_name_room_name_at_begin
device_overrides management with more customizations and filters with regex
fix on sensor: added native_unit_of_measurement and native_value
Added 2 services: update_entities and exec_vimar_sql (to test queries)

If you want, you can merge it to yours :)

Thanks for you support!

davideciarmiello commented:
Hi friend. you would integrate this modification from my branch? Thanks

h4de5 commented:
I did not forget :) - but your PR is quite big :)

I added some comments - don't worry - most of them are for myself.

davideciarmiello commented:
Hi friend. if you need, i have implemented 
- config flow wizard
- devices for entity
- auto add device in area getted from a room name
- added binary_sensor (for switch that i only read)
- cleared a code, creating a coordinator and customizer class
